docs-chef-io/content/supermarket/supermarket_backup_restore.md

```diff
@@ -95,5 +95,5 @@ The restore does not support transferring backups across different versions of S
 For example, to restore a backup in a .dump format, run:
 
 ```bash
-pg_restore --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket_production --verbose supermarket_database_backup.dump
+/opt/supermarket/embedded/bin/pg_restore -U supermarket --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket --verbose supermarket_database_backup.dump
```

Comment:
Hi @sean-horn ,
We have tested the changes you have suggested. It works fine but suggesting a small change from `-U` to `--username` just to maintain similarity for how we mentioned the other attributes for the pg_restore utility and giving the absolute path for the dump file to make sure this command runs from anywhere. Pls commit the suggestion if it looks good.
```suggestion
/opt/supermarket/embedded/bin/pg_restore --username supermarket --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket --verbose ~/supermarket_database_backup.dump
```

Reply:
Hi @sean-horn ,
Can you please work on this PR as per the review comments. Then we can close this.